src/main/java/org/cactoos/experimental/Threads.java

```diff
@@ -188,7 +188,7 @@ public Threads(
      * @param tasks The tasks to be executed concurrently.
      */
     private Threads(
-        final Func<Iterable<Callable<T>>, Iterable<Future<T>>> fnc,
+        final Func<? super Iterable<Callable<T>>, ? extends Iterable<? extends Future<T>>> fnc,
```

Comment:
@rocket-3 I think we are missing some in there, for the first `Iterable`, the `Callable` as well as the `Future`.

Reply:
@victornoel OK, as I see, it's a private ctor., therefore, it should be remained intact, as I think.
So I changed the PR to just puzzle removal commit.
WDYT?

Reply:
@rocket-3 why not indeed